### PR TITLE
Hot reloading refactor

### PR DESCRIPTION
**Pull request by johnno1962**
Hey @zenangst, another tidy of the code to generalise for having a separate Swift Package manager version of InjectionIII. Mostly superficial stuff but I finally got around to porting InjectionClient.mm to Swift. If you spot anything, let me know! 👍

### CODE REVIEW

File: InjectionBundle/SwiftInjection.swift [hunk 1]
```diff
@@ -563,7 +567,8 @@ extension NSObject {
         let object = "@".utf16.first!
         while cls != nil && cls != NSObject.self && cls != NSURL.self {
             let className = NSStringFromClass(cls!)
-            if className.hasPrefix("_") || className.hasPrefix("NS") {
+            if className.hasPrefix("_") || className.hasPrefix("NS") ||
+                className.hasPrefix("WK") {
```
Comment (zenangst) [inline InjectionBundle/SwiftInjection.swift:571]: Is `WK` prefixed classes known to cause issues?

**Review by zenangst — APPROVED** [review]
Couldn't find anything obvious, looks good. Nicely done!

File: InjectionBundle/SwiftInjection.swift (hunk 1 above)
Comment (johnno1962) [reply InjectionBundle/SwiftInjection.swift:571]: Yes, people using WKWebView seems to crash the sweep.

Comment (johnno1962) [reply InjectionBundle/SwiftInjection.swift:571]: Added a "final" change which allows you to inject, injection itself. Thanks for taking a look. Will tweet about "HotReloading" and wait for feedback then merge this.

**Pull request merged**